src/python/WMCore/WMBS/Subscription.py

```diff
@@ -322,26 +317,6 @@ def isCompleteOnRun(self, runID):
         else:
             return False
 
-    def filesOfStatusByRun(self, status, runID):
-        """
-        _filesOfStatusByRun_
-
-        Return all the files in the given subscription and the given run which
-        have the given status.
-        """
-        existingTransaction = self.beginTransaction()
-
-        files = []
-        action = self.daofactory(classname="Subscriptions.Get%sFilesByRun" % status)
```

Comment:
Seangchan, same here, nowhere in WMCore or T0 code we call this method. Meaning, delete it!!!